test/fiber/scheduler.rb

```diff
@@ -180,4 +180,10 @@ def fiber(&block)
 
     return fiber
   end
+
+  def address_resolve(hostname, timeout = nil)
```

Comment:
Do we want a timeout argument here? Scheduler hook is making address resolving non-blocking, so does it make sense to enforce a timeout?

Reply:
No, we shouldn't need to add one, but we might be forced to by the interface.